AdvancedTeleport-Bukkit/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
+Main-Class: 
+Class-Path: paperlib-1.0.4.jar examination-string-1.3.0.jar Configuratio
+ nMaster-API-v2.0.0-BETA-3.jar adventure-nbt-4.10.1.jar examination-api-
+ 1.3.0.jar annotations-13.0.jar
```

Comment:
what is the purpose of this?

Reply:
I believe that file was automatically generated from IntelliJ, I presume it'd be safe to ignore unless you suggest otherwise?

Reply:
Oh yeah then it is basically useless (The MANIFEST.MF file is usually only useful for applications)

Reply:
Cool stuff, I'll pluck it out after work (hooray for the Christmas backlog!!).